### ISSUE TITLE
Negative vs minus

### ISSUE THREAD

**Issue by kelvinsjk** [ronkok/Temml#31]
The typesetting of negative and minus signs seems to be off after = signs

![Screenshot from 2023-10-26 02-22-58](https://github.com/ronkok/Temml/assets/64682375/76ef1eb7-0480-4f58-8436-31659ee8c068)

PS: Thanks for the awesome library

**Comment by ronkok** [comment]
If you want unary operators to have different spacing than binary operators, I'll think you'll have to raise that issue elsewhere.

Maybe at the [MathML-Core issue tracker](https://github.com/w3c/mathml-core/issues/).

**Comment by kelvinsjk** [comment]
Ah, so the issue is with mathml. Thanks for the insight!

**Comment by mk12** [comment]
I think this is an issue with Temml, not MathML. Temml should emit `form="prefix"` for unary operators. For now I'm post-processing the Temml output with this regex:

```js
.replaceAll(
  /(<\/mo>(?:<\/mrow>)?(?:<mrow>)?)<mo>(−|-|\+)<\/mo>/g,
  "$1<mo form=\"prefix\">$2<\/mo>"
)
```

**Comment by ronkok** [comment]
> I think this is an issue with Temml ... Temml should emit `form="prefix"` for unary operators.

@mk12 You are correct! Thank you for pointing that out. I'll have it fixed sometime later this week.

**Comment by xigoi** [comment]
Also happens inside brackets (`(-x)`) and after word operators (`\exp -x`).

**Comment by ronkok** [comment]
Fixed by release v0.10.20.
